tvpeter/automart
# #166466017: User gets all his/her sold ads

#### What does this PR do?

- Enable a user to get his/her sold ads

#### How should this be manually tested?

- Using postman, ensure that one of your ads is purchased and the status is `completed` using sample data.

- log into your account by signing in 

- get your generated token

- make a get request to this endpoint `/api/v1/transactions/sold`

- On the headers tab, set `x-auth` to the value of the copied token

- send your request and wait for the response.

#### What are the relevant pivotal tracker stories?

[#166466017](https://www.pivotaltracker.com/story/show/166466017)
